qutebrowser/components/adblock.py

```diff
@@ -131,6 +131,37 @@ def filter_request(self, info: interceptor.Request) -> None:
                         .format(info.request_url.host()))
             info.block()
 
+    def _read_hosts_line(self, line: str) -> list:
+        """Read hosts from the given line.
+
+        Args:
+            line: The line to read.
+
+        Returns:
+            A list containing valid hosts found
+            in the file.
+        """
+        # Remove comments
+        hash_idx = line.find('#')
+        line = line if hash_idx == -1 else line[:hash_idx]
+
+        parts = line.strip().split()
+        if len(parts) == 1:
+            # "one host per line" format
+            hosts = parts[:1]
```

Comment:
Not sure why I did `hosts = [parts[0]]` here before - `hosts = parts[:1]` works, but wouldn't `hosts = parts` do the same?